src/jvmMain/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvFileWriter.kt

```diff
@@ -36,6 +36,13 @@ class CsvFileWriter internal constructor(
         }
     }
 
+    /**
+     * write one row
+     */
+    override fun writeRow(vararg entry: Any) {
+        writeRow(entry.toList())
```

Comment:
Do you think it is better to specify explicitly the implementation, so it would not look like cyclic?

   override fun writeRow(vararg entry: Any) {
        willWritePreTerminator()
        writeNext(entry.toList())
        willWriteEndTerminator()

        if (writer.checkError()) {
            throw IOException("Failed to write")
        }
    }

Reply:
No, I don't think. Leave this as it is.

Reply:
Thanks for giving your thoughts.